_posts/blog/2019-08-14-Monica's-Final.md

```diff
@@ -0,0 +1,75 @@
+---
+layout: post
+title: Google Summer of Code - Monica's Final Report
+categories: blog
+excerpt: As summer comes to an end, so does GSoC. Monica reflects about her time here.
+image:
+  feature:
+link:
+date: 2019-08-14
+modified:
+share: true
+author: monicayao
+---
+
+## Questions and Answers about this Summer
+
+First, let me bring back an excerpt from my introduction:
+
+> From this summer, I wish to achieve a couple of things: 
+> 1. Learn about the awesome and neat science behind multi-echo fMRI
+> 2. How this library could be advanced in the future, should there be a new publication about denoising ME fMRI data
+> 3. Adding lots and lots of tests to the library so that contributors in the future could extend the library without worries
+> 4. Familiarizing myself with open source, and continue adding onto libraries as a regular contributor!
+
+This is what I wanted to do this summer! It's time to reflect.
+
+### What I wanted to do
```

Comment:
I think this section needs a little bit more about what `tedana` is and why testing is useful.

Just a bit more background for readers who don't already know.

This is where you can add a screenshot of "tests passing" - the output from pytest in the command line.